### PR TITLE
Fixes sewer water entities, adds water entities.

### PR DESCRIPTION
**Pull request by MutantMan2040**
Fixes the sewer water entities to make you submerge in the, added 3 water entities, made all water irradiated. P:

### CODE REVIEW

**Comment by Vonsant** [comment]
@MutantMan2040 i think you need just reopen it to fix Linter error.

**Comment by Vonsant** [comment]
Welp, now this is your error 🤔

File: Resources/Prototypes/Entities/Tiles/water.yml
Comment (Vonsant) [inline Resources/Prototypes/Entities/Tiles/water.yml:1]: WaterIrradiated here

Comment (Vonsant) [reply Resources/Prototypes/Entities/Tiles/water.yml:1]: Instead of irraterradiated water

Comment (MutantMan2040) [reply Resources/Prototypes/Entities/Tiles/water.yml:1]: @Peptide90  This is good now thanks for the help. @Vonsant :)

**Pull request merged**